apps/desktop/src/lib/backend/tauri.ts

```diff
@@ -1,10 +1,37 @@
 import { invoke as invokeIpc, listen as listenIpc } from '$lib/backend/ipc';
 import { getVersion } from '@tauri-apps/api/app';
 import { check } from '@tauri-apps/plugin-updater';
+import { getCurrentWindow } from '@tauri-apps/api/window';
 
 export class Tauri {
 	invoke = invokeIpc;
 	listen = listenIpc;
 	checkUpdate = check;
 	currentVersion = getVersion;
+
+	// Window control methods
```

Comment:
I'm a bit allergic to comments that don't really convey new information. Could we maybe drop this one?